### PR TITLE
Tt 2272 home posts responsive issue

### PR DESCRIPTION
**Pull request by abhishakedahal**
What is done in this PR?

- Fixed home posts responsive issue.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -244,3 +244,10 @@ function envince_body_class( $classes )
 	}
 	return $classes;
 }
+
+// Add the clearfix class to the array of existing classes.
+function add_clearfix_to_article_div( $classes ) {
```
Comment (ashishsthanp) [inline functions.php]: * prefix: https://developer.wordpress.org/plugins/plugin-basics/best-practices/#prefix-everything\

File: functions.php [hunk 2]
```diff
@@ -244,3 +244,10 @@ function envince_body_class( $classes )
 	}
 	return $classes;
 }
+
+// Add the clearfix class to the array of existing classes.
```
Comment (ashishsthanp) [inline functions.php]: * add doc block instead

File: functions.php [hunk 3]
```diff
@@ -244,3 +244,10 @@ function envince_body_class( $classes )
...
+function add_clearfix_to_article_div( $classes ) {
+	$classes[] = 'clearfix';
+	return $classes;
+}
+add_filter( 'post_class', 'add_clearfix_to_article_div' );
```
Comment (ashishsthanp) [inline functions.php]: we might add other classes in the future for other purposes, so use generic callback name like
envince_post_class

**Review by ashishsthanp — CHANGES_REQUESTED** [review]

File: functions.php (hunk 3 above)
Comment (abhishakedahal) [reply functions.php]: done

File: functions.php (hunk 2 above)
Comment (abhishakedahal) [reply functions.php]: done

File: functions.php (hunk 1 above)
Comment (abhishakedahal) [reply functions.php]: done

**Pull request merged**